### ISSUE TITLE
No matching version found for @pigment-css/react@next

### ISSUE THREAD

**Issue by tschaub** [mui/pigment-css#284]
### Steps to reproduce

1. find the manual installation instructions [in the readme.md](https://github.com/mui/pigment-css/blob/7e3e3416f01abab182cff8716ded586de8da4e6c/README.md#manual-installation-1)
2. try `npm install @pigment-css/react@next`
3. see `npm error notarget No matching version found for @pigment-css/react@next.`

### Current behavior

Trying to install either `@pigment-css/react@next` or `@pigment-css/vite-plugin@next` as described in the readme fails with `npm error code ETARGET`.

### Expected behavior

I expected the install to succeed.

### Context

Trying to install dependencies.

### Your environment

<details>
  <summary><code>npx @mui/envinfo</code></summary>
```
  System:
    OS: macOS 14.5
  Binaries:
    Node: 22.9.0 - ~/.nvm/versions/node/v22.9.0/bin/node
    npm: 10.9.0 - ~/.nvm/versions/node/v22.9.0/bin/npm
    pnpm: Not Found
  Browsers:
    Chrome: 130.0.6723.59
    Edge: Not Found
    Safari: 17.5
  npmPackages:
    react: ^18.3.1 => 18.3.1 
    react-dom: ^18.3.1 => 18.3.1 
```
</details>

**Search keywords**: install fails

**Comment by alelthomas** [comment]
Heya @tschaub! Please try installing by removing the `@next` tag:

```
npm install @pigment-css/react
```

You should be able to get v0.0.24. Let me know if you need more help!

**Comment by mnajdova** [comment]
@tschaub would you be up to removing the `next` suffix in the Readme.md instructions?

**Comment by tschaub** [comment]
Doc update in #292.

**Comment by alelthomas** [comment]
Thanks @tschaub!

**Comment by tschaub** [comment]
@alelthomas it looks like #292 has been approved but not yet merged, so the docs issue still exists. Is there anything else I should do to get the fix in #292 in?

**Comment by alelthomas** [comment]
You're good, I will re run the checks and merge 😄 @tschaub
